src/workerd/io/worker-interface.capnp

```diff
@@ -138,6 +177,275 @@ struct Trace @0x8e8d911203762d34 {
 
   truncated @24 :Bool;
   # Indicates that the trace was truncated due to reaching the maximum size limit.
+
+  # The following structs are used only in Streaming Traces.
+
+  # The Onset struct is always sent as the first event in any trace stream. It
+  # contains general metadata about the pipeline that is being traced.
+  struct Onset {
+    accountId @0 :UInt32;
+    stableId @1 :Text;
```

Comment:
Some fields in this struct are internal details which should not be revealed to application code. Why include them here?

Can you please document what the JavaScript API will look like? Your design doc only shows this huge Cap'n Proto definition but doesn't explicitly document how it looks in JavaScript.